cmd/mqtt-bridge/main.go

```
@@ -1,105 +1,49 @@
 	flag.Parse()
 
-	klfPath := os.Getenv("TAPIR_MQTT_BRIDGE_KLF")
-	if klfPath != "" {
-		application.MqttTlsKlfPath = klfPath
+    file, err := os.ReadFile(filename)
+	if err != nil {
+		panic(err)
 	}
 
-	application.Bridges = []app.Bridge{br}
+	yaml.Unmarshal(file, &application)
```

golangci-lint (1.62.2)

Error return value of `yaml.Unmarshal` is not checked

(errcheck)